--- gss-0.0.3/gl/Makefile.am.orig	2003-06-29 18:16:52.000000000 +0200
+++ gss-0.0.3/gl/Makefile.am	2003-08-03 00:14:24.000000000 +0200
@@ -4,11 +4,10 @@
 
 AUTOMAKE_OPTIONS = 1.5 gnits no-dependencies
 
-noinst_LIBRARIES = libfoo.a
+noinst_LTLIBRARIES = libfoo.la
 
-libfoo_a_SOURCES =
-libfoo_a_LIBADD = @LIBOBJS@
-#libfoo_la_LIBADD = @LTLIBOBJS@
+libfoo_la_SOURCES =
+libfoo_la_LIBADD = @LTLIBOBJS@
 EXTRA_DIST =
 BUILT_SOURCES =
 SUFFIXES =
@@ -17,10 +16,10 @@
 DISTCLEANFILES =
 MAINTAINERCLEANFILES =
 
-libfoo_a_SOURCES += gettext.h
+libfoo_la_SOURCES += gettext.h
 
 
 
-libfoo_a_SOURCES += unlocked-io.h
+libfoo_la_SOURCES += unlocked-io.h
 
-libfoo_a_SOURCES += xalloc.h xmalloc.c xstrdup.c
+libfoo_la_SOURCES += xalloc.h xmalloc.c xstrdup.c
--- gss-0.0.3/lib/Makefile.am.orig	2003-06-28 21:58:06.000000000 +0200
+++ gss-0.0.3/lib/Makefile.am	2003-08-03 00:25:51.000000000 +0200
@@ -28,7 +28,7 @@
 
 libgss_la_SOURCES = internal.h gettext.h version.c error.c meta.c asn1.c \
 	context.c cred.c misc.c msg.c name.c obsolete.c oid.c
-libgss_la_LIBADD = @LTLIBINTL@ @SHISHI_LIBS@ ../gl/libfoo.a
+libgss_la_LIBADD = @LTLIBINTL@ @SHISHI_LIBS@ ../gl/libfoo.la
 libgss_la_LDFLAGS = -version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
 
 if KRB5
